app/hooks/useCurrentUserState.ts

```
@@ -1,16 +1,33 @@
 import { useState, useEffect } from "react"
 import { useUser, useSession } from "@clerk/nextjs"
+import { BYPASS_CLERK, DEV_USER_EMAIL } from "@/lib/dev-auth"
 import { useSupabaseClient } from "@/lib/supabase-auth-context"
 import { type User, randomId } from "@/lib/data"
 
 export function useCurrentUserState() {
-  const { user } = useUser()
-  const { session } = useSession()
+  const { user } = BYPASS_CLERK ? { user: null } : useUser()
```

ESLint

[error] React Hook "useUser" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)